nucypher/characters/lawful.py

```diff
@@ -1624,7 +1624,7 @@ def __init__(self, policy_encrypting_key=None, controller: bool = True, *args, *
 
         # Encrico never uses the blockchain, hence federated_only)
         kwargs['federated_only'] = True
-        kwargs['known_node_class'] = Ursula
+        kwargs['known_node_class'] = None
```

Comment:
`None`?

Reply:
Doesn't that seem about right now `Enrico`?

Reply:
Yeah, I guess so.